product_form_purchase_link_oca/models/product.py

```diff
@@ -0,0 +1,22 @@
+# © 2025 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+from odoo import models, fields
+from odoo.tools.float_utils import float_round
+
+class ProductProduct(models.Model):
+    _inherit = 'product.product'
+
+    purchased_product = fields.Float(compute="_compute_purchased_product")
```

Comment:
Replaced field was named `purchased_product_qty`, and `qty` word is key. Our new field could be named instead e.g. `purchased_ever_product_qty`.

But no `purchased_product`, if other developer finds it, he could be very confusing about its meaning (no reference to quantity, or count... and it's a count, not a product!)... fields naming is important.